borgmatic/config/schema.yaml

```diff
@@ -1355,8 +1355,8 @@ properties:
                     - finish
         description: |
             Configuration for a monitoring integration with Healthchecks. Create
-            an account at https://healthchecks.io (or self-host Healthchecks) if
-            you'd like to use this service. See borgmatic monitoring
+            an account at https://healthchecks.io/ (or self-host Healthchecks)
```

Comment:
Is there a reason you've added trailing slashes for these URLs when the the canonical versions don't appear to have trailing slashes?

Some related reading: https://ahrefs.com/blog/trailing-slash/

Reply:
TBH, this is just personal preference from decades of writing Apache configs and stuff like that. Feel free to ignore!

Reply:
Understood. I think I'd prefer to use the canonical URLs for these projects rather than inserting a trailing slash—even if it does look nicer. So would you mind taking these out again? And then I'd be happy to merge the PR so as to get the other fixes!

Reply:
Done.

I also remove one such slash that was there before.